src/navigation/tabNavigator/components/AppHeader.tsx

```diff
@@ -38,6 +25,6 @@ const jsStyles = StyleSheet.create({
     paddingHorizontal: 16,
     width: '100%',
     zIndex: 10,
+    ...Platform.select({ default: { display: 'none' }, web: { display: 'flex' } }),
```

Comment:
Maybe it's better to not render this component on all platforms:

```
if (Platform.OS !== 'web') {
  return null
}
```

Thanks to that we will not run all of this logic that is in return.